pkg/konfluxgen/dockerfile-component.template.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   annotations:
     image.redhat.com/generate: "true"
     appstudio.openshift.io/pac-provision: request
+    build.appstudio.openshift.io/pipeline: '{"name":"docker-build","bundle":"latest"}'
```

Comment:
sorry, I think I asked this already why is this needed ?

Reply:
the pipeline is technically called https://github.com/openshift-knative/hack/blob/d295f2930af63800b286b36b9c90b1f03b746888/pkg/konfluxgen/docker-build-oci-ta.yaml#L8, do they need to match?

Reply:
This annotations is usually added too, when a component gets created via the UI, but leads to "issues" in the UI, when you create them manually (and miss it): https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1717785914904439

Reply:
I remember we had this issue too, so I added it in https://github.com/openshift-knative/serverless-operator/pull/2702

Reply:
But in general, this specifies the builder:
For Index image we have:
```
build.appstudio.openshift.io/pipeline: '{"name":"fbc-builder","bundle":"latest"}'
```
For regular images we have the `docker-build` builder